backend/src/packages/articles/article.controller.ts

```diff
@@ -115,6 +116,28 @@ class ArticleController extends Controller {
           }>,
         ),
     });
+
+    this.addRoute({
+      path: ArticlesApiPath.SHARE,
+      method: 'POST',
+      handler: (options) =>
+        this.share(
+          options as ApiHandlerOptions<{
+            body: { id: number };
+          }>,
+        ),
+    });
+
+    this.addRoute({
+      path: ArticlesApiPath.TOKEN,
+      method: 'POST',
+      handler: (options) =>
+        this.encryptShared(
+          options as ApiHandlerOptions<{
+            body: { token: string };
+          }>,
+        ),
+    });
```

Comment:
I believe both of this routes should be `GET`

Reply:
I changed to POST as I need to get origin header from the request to make share link. However I can use it in the second case. Is it Ok?